src/OneExplorer/OneExplorer.ts

```diff
@@ -405,45 +406,42 @@ input_path=${modelName}.${extName}
       const fstat = fs.statSync(fpath);
 
       if (fstat.isDirectory()) {
-        const childNode = new Node(NodeType.directory, [], vscode.Uri.file(fpath));
+        const dirNode = this.createDirectoryNode(fpath);
 
-        this.searchNode(childNode);
-        if (childNode.childNodes.length > 0) {
-          node.childNodes.push(childNode);
+        if (dirNode) {
+          node.childNodes.push(dirNode);
         }
       } else if (
           fstat.isFile() &&
           (fname.endsWith('.pb') || fname.endsWith('.tflite') || fname.endsWith('.onnx'))) {
-        const childNode = new Node(NodeType.baseModel, [], vscode.Uri.file(fpath));
-
-        this.searchPairConfig(childNode);
+        const baseModelNode = this.createBaseModelNode(fpath);
 
-        node.childNodes.push(childNode);
+        node.childNodes.push(baseModelNode);
       }
     }
   }
 
-  /**
-   * compare paths by normalization
-   * NOTE that '~'(home) is not supported
-   * TODO(dayo) support '~'
-   * TODO(dayo) extract file-relative functions as another module
-   */
-  private comparePath(path0: string, path1: string): boolean {
```

Comment:
(4) Remove unused function comparePath()

This function is no longer used as it uses cfgObj.getDerivedModelsExists now.